manager/filters.py

```diff
@@ -83,7 +83,7 @@ def _filter_cve_by_public_date(query, args):
     return query
 
 
-def _filter_cve_by_rule_presence(query, args):
+def _filter_cve_by_rule_and_presence(query, args):
```

Comment:
Please revert functionality of `_filter_cve_by_rule_presence` function and do the necessary changes in `_filter_system_cve_by_rule` function only.